src/pcm/SoxrResampler.cxx

```diff
@@ -28,20 +28,35 @@
 
 #include <cassert>
 
+#include <cmath>
 #include <string.h>
 
 static constexpr Domain soxr_domain("soxr");
 
 static constexpr unsigned long SOXR_DEFAULT_RECIPE = SOXR_HQ;
 
+/**
+ * The SOXR_RESET_ON_CLEAR flag makes it possible to jump in the position off the current 
+ * track. Applied when using a custom recipe.
+ */
+static constexpr unsigned long SOXR_RESET_ON_CLEAR =
+	2147483648; // 2^31 defined in internal.h of soxr as RESET_ON_CLEAR
+
```

Comment:
This wasn't in your first version of the PR - and I don't like using undocumented features with internal flags. This can break any time.

Reply:
Ok then I hide it by call `soxr_quality_spec` to init `soxr_quality`.